app/src/main/res/values/strings.xml

```diff
@@ -26,4 +26,6 @@
     <string name="lyrics">Lyrics</string>
     <string name="no_data">No Data</string>
     <string name="no_internet">No Internet</string>
+    <string name="name_singer">Singer</string>
+    <string name="avatar">Avatar</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.avatar` appears to be unused
 Unused resources make applications larger and slow down builds.